README.md

```diff
@@ -20,18 +23,23 @@ pqa ask 'How can carbon nanotubes be manufactured at a large scale?'
+> The claim that neural networks have been designed to compute with DNA is supported by multiple sources. The work by Qian, Winfree, and Bruck demonstrates the use of DNA strand displacement cascades to construct neural network components, such as artificial neurons and associative memories, using a DNA-based system (Qian2011Neural pages 1-2, Qian2011Neural pages 15-16, Qian2011Neural pages 54-56). This research includes the implementation of a 3-bit XOR gate and a four-neuron Hopfield associative memory, showcasing the potential of DNA for neural network computation. Additionally, the application of deep learning techniques to genomics, which involves computing with DNA sequences, is well-documented. Studies have applied convolutional neural networks (CNNs) to predict genomic features such as transcription factor binding and DNA accessibility (Eraslan2019Deep pages 4-5, Eraslan2019Deep pages 5-6). These models leverage DNA sequences as input data, effectively using neural networks to compute with DNA. While the provided excerpts do not explicitly mention protein-based neural network computation, they do highlight the use of neural networks in tasks related to protein sequences, such as predicting DNA-protein binding (Zeng2016Convolutional pages 1-2). However, the primary focus remains on DNA-based computation.
 
 ## What is PaperQA2
 
 PaperQA2 is engineered to be the best RAG model for working with scientific papers. Here are some features:
 
-- A simple interface to get good answers with grounded responses that have in-text citations.
-- State-of-the-art implementation including metadata-awareness in document embeddings and LLM-based re-ranking and contextual summarization (RCS).
-- The ability to do agentic RAG which iteratively refines queries and answers.
-- Automatic redundant fetching of paper metadata, including citation and journal quality data from multiple providers.
+- A simple interface to get good answers with grounded responses containing in-text citations.
+- State-of-the-art implementation including document metadata-awareness
+  in embeddings and LLM-based re-ranking and contextual summarization (RCS).
+- Answering process is governed by a language agent which iteratively refines queries and answers.
```

Comment:
"Support for agentic RAG, where a language agent can iteratively refine queries and answers."

Reply:
Done